### PR TITLE
Feat/env files

### PR DESCRIPTION
**Pull request by ryanbasiltrickett**
# Description

Moved application properties files to use a api.env and cli.env files. As well as moved the analysis of the code to the application CI to build up.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: .github/workflows/app-ci.yaml [hunk 1]
```diff
@@ -39,7 +35,15 @@ jobs:
       with:
         languages: ${{ matrix.language }}
 
+    - name: Build API
+      working-directory: ./apps/api
+      run: ./gradlew -q build
+    
+    - name: Build CPI
```
Comment (seanvanwyk-bbd) [inline .github/workflows/app-ci.yaml:42]: CLI*

**Comment by seanvanwyk-bbd** [comment]
Seems good apart from the one spelling item - Do we just put the relevant `.env` file in the same place as the application.properties? It might get a bit hairy if we need to use spring profiles, since there'd be a `.env` for each profile as well.

**Comment by ryanbasiltrickett** [comment]
> Seems good apart from the one spelling item - Do we just put the relevant `.env` file in the same place as the `application.properties`? It might get a bit hairy if we need to use spring profiles, since there'd be a `.env` for each profile as well.

You just need to place an env where you call the build/test/etc commands so there will be an env in api and one in cli

File: .github/workflows/app-ci.yaml (hunk 1 above)
Comment (ryanbasiltrickett) [reply .github/workflows/app-ci.yaml:35]: Fixed

**Review by seanvanwyk-bbd — APPROVED** [review]

**Pull request merged**